src/Microsoft.AspNet.WebHooks.Custom.AzureStorage/WebHooks/AzureWebHookDequeueManager.cs

```diff
@@ -276,9 +276,9 @@ public QueuedSender(AzureWebHookDequeueManager parent, ILogger logger)
                             deleteMessages.Add(message);
                         }
                     }
-                }
-
-                // Loop through the responses to see which messages should be deleted from the queue based on the response statuses.
+                }
+
+                // Loop through the responses to see which messages should be deleted from the queue based on the response statuses.
```

Comment:
This is the opposite of the change in commit c3890ceb0b which is part of #161. VS fixed the mixed line endings but landed with Unix endings even on Windows.

The other files in this PR just had Unix line endings even on Windows with `core.autocrlf` set to `true`.